### ISSUE TITLE
Add option to redirect home to library on mobile website

### ISSUE THREAD

**Issue by shbach** [lawrencehook/remove-youtube-suggestions#72]
I like the option to redirect home to library that is present in the desktop version of this extension and would like the option to also be available when using the mobile website on Firefox Android app.

**Comment by lawrencehook** [comment]
Is it not there already?

I don't have an Android so it's difficult for me to test. Is the option there but not working?

**Comment by coder0107git** [comment]
@shbach Are you able to install this extension on Firefox Android? I tried to figure out how to install it, but couldn't.

**Comment by shbach** [comment]
> Is it not there already?
> 
> I don't have an Android so it's difficult for me to test. Is the option there but not working?

The option doesn't show. 
![Screenshot_20230111-102357](https://user-images.githubusercontent.com/5298745/211888295-5aaaa3f6-2236-411a-a35d-1d3fd8439d0d.jpg)


> Are you able to install this extension on Firefox Android? I tried to figure out how to install it, but couldn't.

You need to be using either the Beta or Nightly version of Firefox on Android. You can follow the instructions here: https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/

**Comment by lawrencehook** [comment]
@shbach That looks like an old version of the extension. Is there a way to update it, perhaps by reinstalling?

**Comment by lawrencehook** [comment]
And thanks for the screenshot! Clearly there's room for improvement on the mobile responsiveness 😅

**Comment by shbach** [comment]
So I checked the version number installed and it said it was the latest version: 
![Screenshot_20230111-170401](https://user-images.githubusercontent.com/5298745/211952477-2d938cf9-6274-4244-9c5e-6ec4e24d2511.jpg)

However, I went ahead and uninstalled and reinstalled and it looks completely different! 
![Screenshot_20230111-171314](https://user-images.githubusercontent.com/5298745/211952615-00d8005a-09a4-4662-b7a1-4b1fb58a5cbf.jpg)
The option is now available and it works so it looks like the issue is with the extension not getting updated. I'll close this issue. Thanks for the help!
